alpha/dotnet-chart/Chart.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: v2
 name: dotnet
-version: 0.0.1
+version: 0.0.2
+appVersion: 8.0.0
+icon: data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDU2IiBoZWlnaHQ9IjQ1NiIgdmlld0JveD0iMCAwIDQ1NiA0NTYiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxyZWN0IHdpZHRoPSI0NTYiIGhlaWdodD0iNDU2IiBmaWxsPSIjNTEyQkQ0Ii8+CjxwYXRoIGQ9Ik04MS4yNzM4IDI5MS4zMzNDNzguMDQ5NiAyOTEuMzMzIDc1LjMwOSAyOTAuMjU5IDczLjA1MiAyODguMTFDNzAuNzk1IDI4NS45MDYgNjkuNjY2NSAyODMuMjg5IDY5LjY2NjUgMjgwLjI1OUM2OS42NjY1IDI3Ny4xNzMgNzAuNzk1IDI3NC41MjkgNzMuMDUyIDI3Mi4zMjVDNzUuMzA5IDI3MC4xMjEgNzguMDQ5NiAyNjkuMDE5IDgxLjI3MzggMjY5LjAxOUM4NC41NTE4IDI2OS4wMTkgODcuMzE5MyAyNzAuMTIxIDg5LjU3NjMgMjcyLjMyNUM5MS44ODcgMjc0LjUyOSA5My4wNDI0IDI3Ny4xNzMgOTMuMDQyNCAyODAuMjU5QzkzLjA0MjQgMjgzLjI4OSA5MS44ODcgMjg1LjkwNiA4OS41NzYzIDI4OC4xMUM4Ny4zMTkzIDI5MC4yNTkgODQuNTUxOCAyOTEuMzMzIDgxLjI3MzggMjkxLjMzM1oiIGZpbGw9IndoaXRlIi8+CjxwYXRoIGQ9Ik0yMTAuMTY3IDI4OS41MTVIMTg5LjIwOUwxMzMuOTk0IDIwMi40MDZDMTMyLjU5NyAyMDAuMjAyIDEzMS40NDEgMTk3LjkxNSAxMzAuNTI4IDE5NS41NDZIMTMwLjA0NEMxMzAuNDc0IDE5OC4wODEgMTMwLjY4OSAyMDMuNTA4IDEzMC42ODkgMjExLjgyN1YyODkuNTE1SDExMi4xNDlWMTcxSDEzNC40NzdMMTg3LjgzOSAyNTYuMDQzQzE5MC4wOTYgMjU5LjU3IDE5MS41NDcgMjYxLjk5NCAxOTIuMTkyIDI2My4zMTZIMTkyLjUxNEMxOTEuOTc3IDI2MC4xNzYgMTkxLjcwOCAyNTQuODU5IDE5MS43MDggMjQ3LjM2NVYxNzFIMjEwLjE2N1YyODkuNTE1WiIgZmlsbD0id2hpdGUiLz4KPHBhdGggZD0iTTMwMC40NDkgMjg5LjUxNUgyMzUuNTYxVjE3MUgyOTcuODdWMTg3LjY5NUgyNTQuNzQ2VjIyMS4yNDlIMjk0LjQ4NVYyMzcuODYxSDI1NC43NDZWMjcyLjkwM0gzMDAuNDQ5VjI4OS41MTVaIiBmaWxsPSJ3aGl0ZSIvPgo8cGF0aCBkPSJNMzkyLjY2NyAxODcuNjk1SDM1OS40NTdWMjg5LjUxNUgzNDAuMjcyVjE4Ny42OTVIMzA3LjE0M1YxNzFIMzkyLjY2N1YxODcuNjk1WiIgZmlsbD0id2hpdGUiLz4KPC9zdmc+Cg==
+home: https://github.com/redhat-developer/s2i-dotnetcore
```

Comment:
I've added some fields here (`appVersion`, `icon`, `home`) to improve the UX.

This is how it looks in OpenShift with these changes:

![image](https://github.com/redhat-developer/redhat-helm-charts/assets/1025424/47d76e7d-2b19-45a0-a84a-d2fdd0f51537)

I didn't find a field that allows to change the 'Dotnet' to '.NET'. It seems to be derived directly from the `name` field.

Reply:
I've changed the `appVersion` from what is shown in this image (`8.0.0`) and removed the patch number (i.e. `8.0`).